### PR TITLE
feat: #619 iMapBC public link

### PR DESCRIPTION
**Pull request by ianliuwk1019**
ref: #619 
- update the text on the about page for the iMpasBC button
'''
View and analyze hundreds of map data layers compiled from across the B.C. Government, and other public sector agencies. This includes the FOM layers, which can be overlayed onto other layers, allowing the user to see FOM features from all published FOMs on one comprehensive map.
'''
- add ghost-button that launches iMapsBC in a new tab (https://maps.gov.bc.ca/ess/hm/imap4m/) at public map details.

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
 - [api](https://fom-21.apps.silver.devops.gov.bc.ca/api)
 - [admin](https://fom-21.apps.silver.devops.gov.bc.ca/admin)
 - [public](https://fom-21.apps.silver.devops.gov.bc.ca/public)

Once merged, code will be promoted and handed off to following workflow run.
 - [Main Merge Workflow](https://github.com/bcgov/nr-fom/actions/workflows/merge-main.yml)

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
 - [api](https://fom-21.apps.silver.devops.gov.bc.ca/api)
 - [admin](https://fom-21.apps.silver.devops.gov.bc.ca/admin)
 - [public](https://fom-21.apps.silver.devops.gov.bc.ca/public)

Once merged, code will be promoted and handed off to following workflow run.
 - [Main Merge Workflow](https://github.com/bcgov/nr-fom/actions/workflows/merge-main.yml)

### CODE REVIEW

File: admin/src/main.ts [hunk 1]
```diff
@@ -77,4 +77,4 @@ bootstrapApplication(AppComponent, {
         ...routesProviders
     ]
 })
-.catch((err) => console.error(err));
\ No newline at end of file
+.catch((err) => console.error(err));
```
Comment (ianliuwk1019) [inline admin/src/main.ts:80]: This is dummy change to trigger pipeline. (nothing change at admin).

File: api/README.md [hunk 2]
```diff
@@ -79,4 +79,3 @@ These are the steps to generate the client library used by the frontend componen
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
```
Comment (ianliuwk1019) [inline api/README.md:81]: This is dummy change on api to trigger pipeline. (nothing change at api)

**Review by MCatherine1994 — APPROVED** [review]

**Comment by MCatherine1994** [comment]
Looks good to me, maybe just let Olga review the UI styling, thanks Ian!!

**Comment by ianliuwk1019** [comment]
Thanks for the quick review @MCatherine1994 !!
Hi @OlgaLiber2 , there is nothing change for the elements spacing this time, only the iMapBC link is added on top of the Map Details. 
Since it looks ok and Julius would like this in this morning, I would just merge this isolated change this time so it will be available for him soon. We can collect other UI adjustments in other ticket together if there are others.
Thanks Olga.

**Comment by OlgaLiber2** [comment]
@ianliuwk1019  sure, can you capture that UI change in a new ticket please? We can add other styling changes to that ticket as well in the future. Thanks for deploying it so promptly. I appreciate it. 

Thanks OIga. Ticket #622 created.

**Comment by webgismd** [comment]
For iMapBC you can add parameters to the url to add the FOM layers on startup.. I can provide the syntax if people want it.

**Comment by OlgaLiber2** [comment]
@webgismd  yes please. Thank you! 
@ianliuwk1019 we can create a new ticket to open FOM layers on startup

**Comment by ianliuwk1019** [comment]
We had quick exploring on the iMapBC and its site help but didn't discover the parameters we had thought. If there is a reference doc for how to, that would be great. Thank you very much @webgismd .

**Comment by webgismd** [comment]
here is an example of the FOM layers added on startup https://maps.gov.bc.ca/ess/hm/imap4m/?catalogLayers=9448,9449,9450,9451,9452,9453 -- there are other parameters we could explore with DataBC... it uses Geocortex Essentials https://docs.vertigisstudio.com/essentials/gvh/

**Pull request merged**